Added: scripts for nightly/rc/release builds (SD card content, Russian voice), Audio normalization, PSV files for TTSAutomate

mr-pokryshkin opened the pull request:
Added: scripts for nightly build (SD card content, Russian voice), system sounds: done, other sounds: partially
Fixed: gTTS issue with unicode (Cyrillic) chars in command params (gTTS module instead of system command)

mr-pokryshkin committed:
Added: scripts for nightly build (SD card content, Russian voice), system sounds: done, other sounds: partially
Fixed: gTTS issue with unicode (Cyrillic) chars in command params (gTTS module instead of system command)

schwabe commented on radio/util/tts_ru.py:
```diff
@@ -0,0 +1,163 @@
+
+for i, (s, f) in enumerate([
+                            (u"Аккумулятор садится!", "lowbat"),
+#                            (u"gear!, up!", "gearup"),
```
Lazyness or a good reason not include the optional sounds? :)

mr-pokryshkin replied:
unfortunately, English texts sounds really weird, if I use Russian voice. So I decided to translate most important sounds, and later, after forum discussion, I'll update optional sounds.

schwabe commented:
Looks good, sound generation worked fine for me. Thanks!

3djc commented:
@bsongis please do not merge before 2.2.1, I would rather deal  with potential server impact in nightlies

schwabe commented:
@3djc then we need to revert the other RU change for time being too, otherwise users will be able to select russian tts but we ship sdcard images without it. I am rather for merging this too.

3djc commented:
No, we have already 5 other languages that are supported, but for wich we do not supply language packs; so it would not be a first

3djc commented:
We could merge it since it applies only to nightly, not either release or rc, but the issue is that  to be run by the server, it would need sdcard version to be incremented, and that would be affecting rc/release too (and that would not generate sdpack for rc/release either). As soon as release is well released, we can merge than, I'll deal with server changes if any, and within one or two day, there will be a russian language pack available as a download. It should make our Russian comrades happy :)

3djc commented:
@mr-pokryshkin Could you complete the translations ? I still see a lot of english in there, then we can merge it

mr-pokryshkin committed:
Fixed: gTTS issue with Unicode (Cyrillic) chars in command params (gTTS module instead of system command), for release candidates and release
Changed: SOX params to normalize the audio
Added: "psv" parameter to export list of audio files into PSV format (see TTSAutomate https://github.com/CaffeineAU/TTSAutomate , to generate voice files using TTS engines from Microsoft, Amazon, Google etc)
Added: some extra voice files for RU

bsongis commented on radio/util/tts_ru.py:
```diff
@@ -0,0 +1,142 @@
+                       u"пятьдесят две", u"шестьдесят две", u"семьдесят две", u"восемьдесят две", u"девяносто две"]):
+    systemSounds.append((s, filename(PROMPT_SYSTEM_BASE + 190 + i)))
+
+for i, (s, f) in enumerate([(u"вольт", "volt0"), (u"вольт", "volt1"),
```
don't we use python3? if yes those `u"..."` are not needed

bsongis commented:
I don't know a single word of russian, so I think it's all good, just the python3 thing

mpaperno commented:
I didn't go through every single string, but the translations (language-wise) looks good to me.   
Очень хорошо!  👍  А теперь Companion, да?  ;-)

bsongis commented:
thanks!

mr-pokryshkin commented:
I'm experienced .NET developer, but I'm newbie python developer. :) This is my first python script. So let me know about best practices or code style, I'll fix my files. If  u"" is optional, let me focus on new features and i'll fix it later.
My roadmap: 
1. Cyrilic fonts (for models naming)
2. Translate Open-Tx messages/menus etc
3. Companion
